scripts/update_repos_version.py

```diff
@@ -0,0 +1,153 @@
+#!/usr/bin/env python
+import os
+import os.path
+import re
+import logging
+import subprocess
+import fileinput
+import shutil
+import click
+import github
+
+LOG_FILENAME = 'bump_repo_version_log.txt'
+logging.basicConfig(filename=LOG_FILENAME,level=logging.INFO)
+
+
+REPOS_TO_CHANGE = (
+    'https://github.com/edx/credentials.git',
+    'https://github.com/edx/course-discovery.git',
+    'https://github.com/edx/ecommerce.git',
+    'https://github.com/edx/programs.git',
+    'https://github.com/edx/edx-analytics-data-api.git',
+    'https://github.com/edx/edx-analytics-dashboard.git',
+    'https://github.com/edx/xqueue.git',
+    'https://github.com/edx/edx-platform.git',
+)
+
+# Here's where we'll clone the repos.
+TEMP_DIRECTORY = 'git_tmp'
```

Comment:
We should make a new random tmp dir using [mkdtemp](https://docs.python.org/2/library/tempfile.html#tempfile.mkdtemp).

This way we can run it multiple times without having to clean out the temp directory manually.